### PR TITLE
Fix ci and add release action workflow

### PR DESCRIPTION
**Pull request by yixinglu**
This PR depends on vesoft-inc/nebula-graph#1165

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+          - arm
+        exclude:
+          - os: darwin
+            arch: arm
+    steps:
+      - uses: actions/checkout@v2
+      - name: setup-go 1.14
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.14
```
Comment (jievince) [inline .github/workflows/release.yaml:30]:
```
      matrix:
        os:
          - linux
          - darwin
          - windows
        arch:
          - amd64
          - arm64
          - arm
        exclude:
          - os: darwin
            arch: arm
          - os: windows
            arch: arm64
    steps:
      - uses: actions/checkout@v2
        with:
          fetch-depth: 1
      - name: setup-go 1.16.4
        uses: actions/setup-go@v1
        with:
          go-version: 1.16.4
```
Maybe we can use this configuration. It  has been tested via nebula-console

**Review by Aiee — DISMISSED** [review]

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+              output_name+='.exe'
+          fi
+          echo "::set-output name=package_name::$output_name"
+      - name: build
+        env:
+          GOOS: ${{ matrix.os }}
+          GOARCH: ${{ matrix.arch }}
+        run: |
+          cd cmd
+          go build -o ../${{ steps.package-name.outputs.package_name }}
```
Comment (jievince) [inline .github/workflows/release.yaml:45]: The built binary of nebula-importer is a dynamic executable. Could specify ```CGO_ENABLED=0``` when use go build to make it a static binary
```
ldd ./nebula-importer
        linux-vdso.so.1 (0x00007ffd5b58f000)
        libpthread.so.0 => /usr/lib64/libpthread.so.0 (0x00007fe85a9d3000)
        libc.so.6 => /usr/lib64/libc.so.6 (0x00007fe85a80d000)
        /lib64/ld-linux-x86-64.so.2 (0x00007fe85aa10000)
```

File: .github/workflows/release.yaml (hunk 1 above)
Comment (Aiee) [reply .github/workflows/release.yaml:30]: ` go-version: 1.16.4`
We should require a lower version of Golang like 1.13

Comment (jievince) [reply .github/workflows/release.yaml:30]: It's just used when packaging releases in actions. When compiling it by source code, the needed golang version is still 1.13

File: .github/workflows/release.yaml (hunk 2 above)
Comment (yixinglu) [reply .github/workflows/release.yaml:49]: Good Catch! Fixed.

File: .github/workflows/release.yaml (hunk 1 above)
Comment (yixinglu) [reply .github/workflows/release.yaml]: Fixed.

File: .github/workflows/release.yaml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        exclude:
+          - os: darwin
+            arch: arm
+          - os: windows
+            arch: arm64
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v1
+        with:
+          go-version: 1.13
```
Comment (jievince) [inline .github/workflows/release.yaml:32]: darwin/arm64 is only supported after golang 1.16

Comment (yixinglu) [reply .github/workflows/release.yaml]: ok, fixed. Thx.

**Review by jievince — APPROVED** [review]

**Pull request merged**